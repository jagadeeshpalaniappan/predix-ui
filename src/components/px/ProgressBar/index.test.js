import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import ProgressBar from './';

describe('ProgressBar', () => {
  test('should render', () =>{
    const tree = renderer.create(<ProgressBar value={75}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
