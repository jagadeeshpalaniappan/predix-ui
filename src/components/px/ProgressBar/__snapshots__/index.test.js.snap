// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar should render 1`] = `
.c2 {
  background-color: var(--px-progress-bar-fill-color,black);
  height: var(--px-progress-bar-height,5px);
  -webkit-transition: -webkit-transform 1s cubic-bezier(.78,.13,.16,.87);
  -webkit-transition: transform 1s cubic-bezier(.78,.13,.16,.87);
  transition: transform 1s cubic-bezier(.78,.13,.16,.87);
  -webkit-transform-origin: left center;
  -webkit-transform-origin: left center;
  -ms-transform-origin: left center;
  transform-origin: left center;
  -webkit-transform: scaleX(0);
  -webkit-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  overflow: hidden;
  -webkit-transform: scaleX(0.75);
  -ms-transform: scaleX(0.75);
  transform: scaleX(0.75);
}

.c1 {
  background-color: var(--px-progress-bar-background-color,lightgray);
  height: var(--px-progress-bar-height,5px);
}

.c0 {
  height: var(--px-progress-bar-height,5px);
}

<div
  aria-valuemax={100}
  aria-valuemin={0}
>
  <div
    className="c0"
  >
    <div
      className="c1"
    >
      <div
        className="c2"
        value={75}
      />
    </div>
  </div>
</div>
`;
